packages/protocol/contracts/common/MetaTransactionWalletDeployer.sol

```diff
@@ -44,36 +31,9 @@ contract MetaTransactionWalletDeployer is
 
   /**
      * @notice Used in place of the constructor to allow the contract to be upgradable via proxy.
-     * @param initialDeployers a list of addresses that are allowed to deploy wallets
      */
-  function initialize(address[] calldata initialDeployers) external initializer {
+  function initialize() external initializer {
```

Comment:
Does this contract still need to be ownable? Does it need to be initializable?

Reply:
Nope good callout 👍